python/pysrc/kaskada/_ffi.pyi

```diff
@@ -34,6 +34,12 @@ class Expr:
         session: Session,
         value: int | float | str | None,
     ) -> Expr: ...
+    @staticmethod
+    def literal_datetime(
```

Comment:
Why is this called `_datetime` when it creates a `timedelat`? What will happen in the future if we want to support datetime?

Reply:
That is because it definitely should not be called _datetime